kong/plugins/prometheus/enterprise/exporter.lua

```diff
@@ -0,0 +1,110 @@
+local kong = kong
+local ngx = ngx
+local sub = string.sub
+local floor = math.floor
+
+local metrics = {}
+
+
+local function init(prometheus)
+  metrics.license_errors = prometheus:counter("enterprise_license_errors",
+                                              "Errors when collecting license info")
+  metrics.license_signature = prometheus:gauge("enterprise_license_signature",
+                                              "Last 32 bytes of the license signautre in number")
+  metrics.license_expiration = prometheus:gauge("enterprise_license_expiration",
+                                                "Unix epoch time when the license expires, " ..
+                                                "the timestamp is substracted by 24 hours "..
+                                                "to avoid difference in timezone")
+  metrics.license_features = prometheus:gauge("enterprise_license_features",
```

Comment:
switch to label == feature with values of ee_plugins and write_admin_api and a value of 0 or 1 i.e.
`enterprise_license_features{feature="ee_plugins"}: 1`
`enterprise_license_features{feature="write_admin_api"}: 1`